### PR TITLE
SM-5718 add osgi bundle for Mozilla Rhino 1.7.15

### PR DESCRIPTION
**Pull request by jherkel**
jira task https://issues.apache.org/jira/browse/SM-5718

### CODE REVIEW

File: rhino-1.7.15/pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
     <groupId>org.apache.servicemix.bundles</groupId>
     <artifactId>org.apache.servicemix.bundles.rhino</artifactId>
-    <version>1.7.14_3-SNAPSHOT</version>
+    <version>1.7.15-SNAPSHOT</version>
```
Comment (jbonofre) [inline rhino-1.7.15/pom.xml:33]: The version should be `1.7.15_1-SNAPSHOT`.

File: rhino-1.7.15/pom.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
 
     <properties>
         <pkgGroupId>org.mozilla</pkgGroupId>
-        <pkgVersion>1.7.14</pkgVersion>
+        <pkgVersion>1.7.15</pkgVersion>
```
Comment (jbonofre) [inline rhino-1.7.15/pom.xml:47]: We should update to bundles-pom 16 and define the source version property.

**Review by jbonofre — CHANGES_REQUESTED** [review]
I can do the changes for you if you want.

**Comment by jherkel** [comment]
> I can do the changes for you if you want.

If it is ok for you I would appreciate...

**Pull request closed without merging**